### ISSUE TITLE
Add 'Boomette' to Hard Rain

### ISSUE THREAD

**Issue by True-Boredom** [Tsuey/L4D2-Community-Update#307]
### Suggestion

Dark Carnival, Swamp Fever, and The Parish are all able to spawn female boomers, but for some reason they were ommitted from Hard Rain.

Hard Rain is the only campaign set after their introduction in Dark Carnival that doesn't allow them to spawn, so I think they should be added, which would also be an easy thing to do.

**Comment by Tatertot10** [comment]
I agree, she spawned on the old 360 version. But i think she should be removed from the parish.
